### PR TITLE
EOS-18738: Unskip non-unit test cases

### PR DESCRIPTION
**Pull request by rahuljyoti65**
Signed-off-by: Rahul Jyoti <rahul.jyoti@seagate.com>

### CODE REVIEW

File: cli/src/code_coverage/prvsnr-coverage.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def install_coverage():
         "yum install -y gcc cpp python3-devel"
     )
     _run(
-        "pip3 install -U api/python"
+        "pip3 install -U api/python[test]"
```
Comment (andkononykhin2) [inline cli/src/code_coverage/prvsnr-coverage.py:35]: please compare `test` extra in setup.py with test-requirements.txt and add missed dependencies in the former. So the next step (`pip3 install -r test-requirements.txt`) can be removed.

**Review by andkononykhin2 — CHANGES_REQUESTED** [review]
@rahuljyoti65 That is pretty well. Thanks. Just one improvement ask to address. Please

File: cli/src/code_coverage/prvsnr-coverage.py (hunk 1 above)
Comment (rahuljyoti65) [reply cli/src/code_coverage/prvsnr-coverage.py:35]: Done !

File: api/python/setup.py [hunk 2]
```diff
@@ -40,6 +40,11 @@
     'pytest-xdist==1.29.0',
     'pytest-timeout==1.3.4',
     'pytest-mock==3.1.0',
+    'attrs==19.1.0',
```
Comment (andkononykhin2) [inline api/python/setup.py:43]: no, please remove that, attrs is [part of our repo](https://github.com/Seagate/cortx-prvsnr/tree/pre-cortx-1.0/api/python/provisioner/vendor/attr) actually

File: api/python/setup.py [hunk 3]
```diff
@@ -40,6 +40,11 @@
...
+    'PyYAML==5.1.2',
```
Comment (andkononykhin2) [inline api/python/setup.py:44]: please remove, PyYAML is one of our main dependencies already

File: api/python/setup.py [hunk 4]
```diff
@@ -40,6 +40,11 @@
...
+    'salt==2019.2.0',
```
Comment (andkononykhin2) [inline api/python/setup.py:45]: the same for salt

**Review by andkononykhin2 — CHANGES_REQUESTED** [review]

File: api/python/setup.py (hunk 2 above)
Comment (rahuljyoti65) [reply api/python/setup.py]: Done

File: api/python/setup.py (hunk 3 above)
Comment (rahuljyoti65) [reply api/python/setup.py]: Done.

File: api/python/setup.py (hunk 4 above)
Comment (rahuljyoti65) [reply api/python/setup.py]: Done

**Review by andkononykhin2 — APPROVED** [review]

**Pull request merged**